api/TimeRanges.json

```diff
@@ -29,10 +29,10 @@
             "version_added": true
           },
           "safari": {
-            "version_added": true
+            "version_added": "5.1"
```

Comment:
Hmm, so https://mdn-bcd-collector.appspot.com/tests/api/TimeRanges only tests the prototype, and api.HTMLMediaElement.buffered which is a `TimeRanges` is supported since Safari 3.2. Can you add custom tests for this?